static/sass/_pattern_blog-card.scss

```diff
@@ -0,0 +1,40 @@
+@mixin blog-p-card {
+  .p-card--post {
```

Comment:
Could you just use the [raw content bleed Vanilla](https://vanillaframework.io/docs/patterns/card#content-bleed) pattern, please?

Reply:
The raw one causes things not to be aligned. The footer will not stick at the bottom it will just be below the text and the border does not align correctly in the footer.